apis/types/peer_create_request.go

```diff
@@ -85,64 +85,48 @@ func (m *PeerCreateRequest) Validate(formats strfmt.Registry) error {
 	return nil
 }
 
-func (m *PeerCreateRequest) validateIP(formats strfmt.Registry) error {
```

Comment:
Please fulfil the information in PR Description according to the outline and revise your commit log. If you feel confused, please read guide doc: https://github.com/dragonflyoss/Dragonfly/blob/master/CONTRIBUTING.md

Reply:
A better choice: describe clearly why did you modify those logics. It is beneficial for reviewers.

Reply:
Okay, let me read the guide.

Reply:
Before you submit your code, you can run some commands, such as `make unit-test or make build and so on` which can help you test your code in your local developing environment.

Reply:
ok